contrib/python/pypodman/pypodman/lib/podman_parser.py

```diff
@@ -48,6 +48,18 @@ def __init__(self, **kwargs):
 
         super().__init__(**kwargs)
 
+    def add_flag(self, *args, **kwargs):
+        """Add flag to parser."""
+        flags = [a for a in args if a[0] in self.prefix_chars]
```

Comment:
As long as you're doing a new PR, what would you think of removing `flags` entirely and just using `args`? I haven't examined every possible scenario, but I'm fairly sure that this is a NOP: there's no reason to call `add_flag` with a non-option (positional) argument, is there?